src/components/look/tag-filter/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { Select } from 'antd';
+import { Option } from 'antd/lib/mentions';
+import styled from 'styled-components';
+
+import { P } from '@src/component/atoms';
+
+import STYLE_TAG from '@src/data/style-tags.json';
+
+function TagFilter({ idSetter }) {
+  function onChange(value, option) {
```

Comment:
```suggestion
  function handleSelectChange(value, option) {
```

이벤트 핸들러는 handle이라는 이름으로 시작하도록 네이밍 해주세요~

Reply:
e6a6f5e